### ISSUE TITLE
Links with phone number did not work in frontend

### ISSUE THREAD

**Issue by sohamakl** [trendschau/typemill#472]
Links with phone number in the format tel:+492118755 did not work in the frontend but supported in the ebook plugin.

**Comment by sohamakl** [comment]
Gibt es hierzu schon einen Lösungsansatz?

**Comment by trendschau** [comment]
I tested this and it works for me even in mobile devices. Are there any more details what does not work in detail?

**Comment by sohamakl** [comment]
I was probably not precise enough. If I use http:// or https:// without markdown formatting, it is shown as a link. With tel: it does not work. Here is an example:

```
Content

tel:+492118755

mobile: [+4917212345](tel:+4917212345)

https://test.com

[:address:]

Typemill Inc.  
Millerstreet 12
11223 Bread-City

https://test.de

tel:+49123456789

mobile: [+4917212345](tel:+4917212345)

[:/address:]
``` 

![Image](https://github.com/user-attachments/assets/9f04bd83-39e9-4322-b1a3-9c319be876ef)

**Comment by trendschau** [comment]
Yes, that is autolinking and the parsedown library that I use seems to support this feature only for links starting http and https, not for other like tel or mailto. You can test it here: https://parsedown.org/demo

But with explizit linking in markdown it looks fine?

**Comment by sohamakl** [comment]
It looks like the parsedown project is currently lacking a bit of support, which is a shame. I hope this will not become a problem for Typemill in future php versions.

**Comment by trendschau** [comment]
Yes, but I am quite happy that the initiator of parsedown updated his project recently again. There are more libraries out there but parsedown is very widespread and convenient to work with
